### ISSUE TITLE
Breaking Changes

### ISSUE THREAD

**Issue by ThibaudDauce** [Cosmian/findex#14]
When the next breaking change is ongoing, check :BreakingChange inside the code

**Comment by tbrezot** [comment]
Can this issue be closed ?

**Comment by ThibaudDauce** [comment]
It's not done as far as I remember… We should reserve the `1 ` error code for unknown error I think and set a different one for buffer too small.

**Comment by tbrezot** [comment]
I agree FFI error codes could be improved. I had the same thought while working on the CoverCrypt FFIs. It would be great to have the same conventions for both repos. This will be easier with the creation of CloudproofRust.

Btw, I think this issue can be moved to the CloudproofRust repo once #22 is merged (or even before if you want).

**Comment by tbrezot** [comment]
I close this issue since the interface has been moved to the CloudproofRust repo. A corresponding issue is opened in the CloudproofRust repo ([link](https://github.com/Cosmian/cloudproof_rust/issues/10)).
